src/RestaurantButton.js

```diff
@@ -0,0 +1,9 @@
+const RestaurantButton = ({ onClickHandler }) => {
```

Comment:
```suggestion
const RestaurantButton = ({ handleClick }) => {
```
More common to see React devs write `handleClick` but not a huge deal.